src/licensedcode/data/rules/gpl-3.0-plus_22.yml

```diff
@@ -0,0 +1,3 @@
+licenses:
+    - gpl-2.0
+notes: no version is specified and that the original archive has both a GPL 2.0 and 3.0 texts

```

Comment:
Please add a bit more to the note including a link to the original sprm and the fact that ansiknr is originally GPL-2.0-licensed.